sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -0,0 +1,100 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the Key Vault Certificate llong running operations.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/internal/client_options.hpp>
+
+#include "azure/keyvault/certificates/certificate_client_models.hpp"
+#include "azure/keyvault/certificates/dll_import_export.hpp"
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
```

Comment:
I don;t think you need to include `http.hpp`, `operation_status.hpp`, `client_option.hpp`.
Please, review includes carefully in all public headers. The problem is that if we include extra, it could be a breaking change to remove a header include post-GA.